todo_app/data/test_view_model.py

```diff
@@ -0,0 +1,30 @@
+from todo_app.data.item import Item
+from todo_app.data.trello_items import get_items
+from todo_app.data.view_model import ViewModel
+
+
+def test_view_model_done_property():
+    # Arrange
+    items = [
+        Item('1','My card','Open'),
+        Item('2','My 2nd card','Closed')]
```

Comment:
_Really_ minor, but I'm a fan of moving this closing bracket onto a new line, so I can see at a glance that the opening & closing brackets for the `items` list match:
```suggestion
    items = [
        Item('1','My card','Open'),
        Item('2','My 2nd card','Closed')
    ]
```